README.rst

```diff
@@ -78,6 +78,50 @@ This example demonstrates the library with the single built-in DotStar on the
     pixels = adafruit_dotstar.DotStar(board.APA102_SCK, board.APA102_MOSI, 1)
     pixels[0] = (10, 0, 0)
 
+
+This example demonstrates the library with the DotStar Feather Wing and bounces Blinka.
+
+`Feather M4 Express <https://www.adafruit.com/product/3857>`_ and
+`DotStar FeatherWing <https://www.adafruit.com/product/3449>`_.
+
+.. code-block:: python
```

Comment:
I think this example would be better to have in a separate example file. The example in the readme should be short and simple. Other examples can do fancier things.